### ISSUE TITLE
The minimap becomes larger than normal

### ISSUE THREAD

**Issue by xsimaa** [Seekiii/nv_loadingscreen#2]
The minimap becomes larger than normal, basically it enlarges, this problem already happened to me with another loading screen that the creator was able to solve for me (I don't know what he did to fix it). I don't know what could cause the problem. 

The loading screen works very well and is very good.

That problem happens to me.

Does it happen to anyone else?

Sorry if I don't understand something, English is not my main language.

![imagen_2024-10-08_234239616](https://github.com/user-attachments/assets/88a467fa-e1cf-4bf8-adf3-b8e7e4e7d5d3)

**Comment by Seekiii** [comment]
Our loading screen has absolutely nothing to do with the map. Out of 1000 downloads, you're the first to experience this issue.

Can you check if some other script might have increased the map size? In our code, which is open-source and available for you to check, we have no connection with the minimap.

Try stop the loading screen script and see if the minimap returns to normal, then try start it again.

**Comment by xsimaa** [comment]
I found the error.
 
There is something happening between the hud and the loading screen that causes this to happen. 

I removed the hud from the server leaving the loading screen and the map was fixed. 

I put the hud on and take out the loading screen and it also fixes.

I have to see what it could be from the hud that is interfering with the loading screen, it is a very strange situation because the hud has nothing in common with a loading screen.

Then when I restart the hud script it is also fixed.

I don't know what causes it, I'll have to investigate.

Thank you so much.
